components/android/app/build.gradle

```diff
@@ -39,11 +40,12 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "org.codingindia.travel_safe"
-        minSdkVersion 16
+        minSdkVersion 21
```

Comment:
@tanmayrauth  and @ankitshaw  could you guys please confirm what is the min API version required here.

Reply:
As I have mentioned earlier I was facing issue with multiDex, so I tried to change the min version as stated by a person on GitHub issue.

Reply:
If we can get the confirmation regarding the decided min API version than it could be easier for us. @tanmayrauth  and @ankitshaw your input is required here.

Reply:
That would do.